portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I measure about 5 feet 10 inches!','I am really good at MarioKart and Just Dance!',
+    'Some of my favorite movies are Now You See Me and Ratatouille!',"I have a 6 month Cane Corso named Mamba!",
+    "I am allergic to cats:(!"];
```

Comment:
You use single quotes for the first 3 entries of this array, and double quotes for the second 2 entries - keep it consistent and use just one. With javascript, either one works, at Google we do single quotes, but that might vary from company to company, so just pick your favorite :)